min/lib/Minify/Controller/MinApp.php

```diff
@@ -136,13 +136,14 @@ public function setupSources($options) {
                 $base = '/';
             }
             $allowDirs = array();
+            $document_root = realpath($_SERVER['DOCUMENT_ROOT']);
```

Comment:
thanks a lot!

2016-05-12 19:31 GMT+02:00 Steve Clay <notifications@github.com>:

> In min/lib/Minify/Controller/MinApp.php
> <https://github.com/mrclay/minify/pull/527#discussion_r63062361>:
>
> > @@ -136,13 +136,14 @@ public function setupSources($options) {
> >                  $base = '/';
> >              }
> >              $allowDirs = array();
> > +            $document_root = realpath($_SERVER['DOCUMENT_ROOT']);
>
> My latest commit
> <https://github.com/mrclay/minify/commit/de39966e9c2ba90b7e400f92b867f6592e34046f>
> handles this for the upcoming 3.0. In 2.x, you should manually set
> $_SERVER['DOCUMENT_ROOT'].
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/mrclay/minify/pull/527/files/42d9041d2d744995c10c0c3bba34648df9864211#r63062361>
>